suite/regress/arm_v5.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+
+# Test ARMv5 mode assembles only supported features.
+
+# Author: Andrew O'Brien
+
+from keystone import (Ks, KsError, KS_ARCH_ARM, KS_MODE_ARM, KS_MODE_V5)
+import regress
+
+class TestARM(regress.RegressTest):
+    asm = b"""
+    MOVT r0, #0
+    """
```

Comment:
I don't think there are any ARMv5 instructions that don't also exist in newer versions, for backwards compatibility. Only deprecation warnings are issued in the migration manuals in favor of newer features, but no breaking changes are made.